feature-utils/poly-sparql/src/query-templates.js

```diff
@@ -0,0 +1,19 @@
+import { polyNs, rdfNs, rdfsNs, xmlNs } from "./namespaces";
+
+export function sparqleInsertTemplate({ graph, triples }) {
+    return `PREFIX poly: ${polyNs} PREFIX rdf: ${rdfNs} PREFIX rdfs: ${rdfsNs} PREFIX xml: ${xmlNs} INSERT DATA { ${
+        (graph && `GRAPH ${graph} {`) || ""
+    } ${triples} } ${(graph && "}") || ""}`;
+}
+
+export function sparqleDeleteTemplate(graph, triples) {
+    return `\
+    PREFIX poly: ${polyNs} \
+    PREFIX rdf: ${rdfNs} \
+    PREFIX rdfs: ${rdfsNs} \
+    PREFIX xml: ${xmlNs} \
```

Comment:
Same  here. Up to this line, it's a constant, and as far as I can tell, it's the same constant. You probably need to define it only once.